### PR TITLE
Update subbridge doc for sygma launch

### PR DESCRIPTION
**Pull request by tolak**
-

### CODE REVIEW

File: other-products/subbridge/technical-details.md [hunk 1]
```diff
@@ -53,19 +53,22 @@ The registration of SubBridge assets is mainly divided into two parts:
...
+The second part is to enable the corresponding EVM bridge. This part is only for the asset settings that want to carry out the cross-chain requirement from Phala to the EVM chain. In SubBridge, the same asset can enable both SygmaBridge-based bridges and ChainBridge-based bridges. In practice, users are always willing to choose solutions with lower fees.
 
 The steps to do the registration stuff are as follow:
 
-*   Step1, we schedule a call of `pallet-registry::forceRegisterAsset` with given registration informations. When the council enacted the call, an asset instance will be created by `pallet-assets`, and some extra registration information will be saved in `pallet-registry`.
+*   Step1, we schedule a call of `pallet-registry::forceRegisterAsset` with given registration informations. When the technical committee enacted the call, an asset instance will be created by `pallet-assets`, and some extra registration information will be saved in `pallet-registry`.
```
Comment (h4x3rotab) [inline other-products/subbridge/technical-details.md]: Are you sure it's the technical committee?

Comment (tolak) [reply other-products/subbridge/technical-details.md]: Yeah, it's Council, fixed already

File: other-products/subbridge/supported-assets.md [hunk 2]
```diff
@@ -1,13 +1,11 @@
...
+* Assets on the Phala network
```
Comment (HashWarlock) [inline other-products/subbridge/supported-assets.md:9]: Is it worth adding the assets in a table so users don't need to go to Polkadot.js.org?

Comment (tolak) [reply other-products/subbridge/supported-assets.md:9]: I personally think polkadot.js.org is straightforward, here I'd like to only mention assets that enabled solo EVM chain transferring. So far we only have PHA and ZLK, if more assets exist, I will use a table

**Pull request merged**